core/arch/arm/mm/core_mmu.c

```diff
@@ -959,6 +959,10 @@ static void check_pa_matches_va(void *va, paddr_t pa)
 		return;
 	}
 #ifdef CFG_WITH_PAGER
+	if (v >= CFG_TEE_LOAD_ADDR && v < core_mmu_linear_map_end) {
+		TEE_ASSERT(v == pa);
```

Comment:
minor: why TEE_ASSERT() and not assert() ?
note: core_mmu.c uses assert() but in check_pa_matches_va() that uses TEE_ASSERT().